modules/test/playwright/tests/dynamic-data-mapping-form-web/main/formFields.spec.ts

```diff
@@ -64,6 +56,51 @@ test.describe('Manage fields through Form Preview page', () => {
 		await newTabPage.close();
 	});
 
+	test('duplicating field with evaluation rules has correct behavior', async ({
+		formBuilderPage,
+		formBuilderSidePanelPage,
+		page,
+	}) => {
+
+		// We're creating a form with repeatable and conditional text field, then duplicating it from their input
+		// field and clicking on duplicate button to trigger evaluate (onBlur event) and duplicate
+		// (onClick event) events at the same time. For now we're first triggering evaluate event and then duplicate event
+
```

Comment:
In my opinion the test flow is very clear in the code. Is this comment really necessary?